website/README.md

```diff
@@ -2,32 +2,31 @@
 ---
 
 - [Prerequisites](#prerequisites)
-  - [Docker](#docker)
   - [OSX, Linux, Windows](#osx-linux-windows)
+  - [Docker](#docker)
```

Comment:
Could we move this link to the “Running the stack” heading as that’s where it leads yet it’s under the prerequisites section